### DISCUSSION TITLE
How to upload c++ desktop app which developed by WindowsAppSDK to microsoft app store?

### DISCUSSION THREAD

**Discussion by shelllet** [microsoft/WindowsAppSDK#1918] [Q&A] [answered]
i purchase microsoft develop acount, but i don't know how to upload package app by visual studio?

i package and publish app by visual studio, and assoicate app with store, but  Visual studio told me: 
```
2>MakeAppx : error : Error info: error C00CE020: App manifest validation error: The app manifest must be valid as per schema: Line 39, Column 4, Reason: Required attribute 'PhonePublisherId' is missing.
```
what is 'PhonePublisherId`?

when i found `PhonePublisherId` and fill attribute.
![image](https://user-images.githubusercontent.com/16729017/146530547-535ec8c4-e96f-461d-80f5-11064197f94a.png)

next:
![image](https://user-images.githubusercontent.com/16729017/146524720-4572d216-bef9-4973-8a62-0545f83ebe4a.png)

next:

![image](https://user-images.githubusercontent.com/16729017/146524926-6bb7025a-0adb-4066-8901-4f17e89f2eb1.png)

what is `Tenant Id`, and `Client Id` ? 

when id found this:

![image](https://user-images.githubusercontent.com/16729017/146527541-4fd49074-e97b-4075-8910-98febe287bb1.png)

i use `Secret ID` as `Client Id` and  Value as `Client Key`, but config failed.

![image](https://user-images.githubusercontent.com/16729017/146527935-9917680e-9a96-4d31-89f8-f25e32e0c6b4.png)

is there any tutorial doc to tell us  how to upload app to microsoft app store? Why this process is so complicated?

and what is this ? i'm crazy!
![image](https://user-images.githubusercontent.com/16729017/146529387-eb624deb-ef00-406f-be0d-5f184a59b9bd.png)

**Comment by andrewleader** [comment, marked as answer]
Hi @shelllet, since this is a Store publishing question, please contact the Partner Center support team by using the Support page on the bottom left of your Partner Center account.

![image](https://user-images.githubusercontent.com/13246069/146582775-c6400d3b-0c33-41c9-aa4d-c4bccdce67e7.png)

**Reply by shelllet** [reply]
Ok , thks

**Reply by sowenzhang** [reply]
@shelllet actually, you just need to manually remove the Identity line from the package manifest. 
I encountered the same issue, after removing that line, it's fine.
